### ISSUE TITLE
support init containers

### ISSUE THREAD

**Issue by jseguillon** [aylei/kubectl-debug#14]
Supporting init containers would be great for troubleshooting.

I have some troubleshooting to do on [mongodb-replicaset](https://github.com/helm/charts/tree/master/stable/mongodb-replicaset), getting locked with init containers. So tried this beautiful tool but : 

```bash
[test@test ~]$ kubectl debug --namespace test test-mongodb-replicaset-1  
container mongodb-replicaset id not ready
[test@test ~]$ kubectl debug --namespace test test-mongodb-replicaset-1 -c bootstrap
cannot find specified container bootstrap
```

kubectl can see this bootsrap (still running, that is the problem) running :  
```bash
[test@test ~]$ kubectl logs test-mongodb-replicaset-1 -n test -c bootstrap -f 
2019/02/21 11:05:13 Peer list updated
... waiting for new logs that will never come...
```

Extract from describe : 
```bash 
kubectl -n test describe pod test-mongodb-replicaset-1 
Name:               test-mongodb-replicaset-1
Namespace:          test
Priority:           0
PriorityClassName:  <none>
Node:               xxx
Start Time:         Thu, 21 Feb 2019 11:05:09 +0000
Labels:             app=mongodb-replicaset
                    controller-revision-hash=test-mongodb-replicaset-647db4c6c4
                    release=test
                    statefulset.kubernetes.io/pod-name=test-mongodb-replicaset-1
Annotations:        <none>
Status:             Pending
IP:                 10.233.88.14
Controlled By:      StatefulSet/test-mongodb-replicaset
Init Containers:
  copy-config:
    Container ID:  docker://f6685231928ba6a843dd348f6cdd602f178aa5e132194dc8aaa44b8058d02c21
   ...
    State:          Terminated
  install:
    Container ID:  docker://618fcfe64504376ece61dfc481641921f0bf1226a6b1394503f8622e0fed0cd9
    State:          Terminated   
   ...
  bootstrap:
    Container ID:  docker://34e529ecd86058e1c9e45a1a50f51d610ec65181c0bdfab17b46010c47083142
    Image:         mongo:3.6
    Image ID:      docker-pullable://mongo@sha256:89822fa6161c2ed77e73fb717f189e6ce5a95cb752fe2021508daeee366f9b69
    Port:          <none>
    Host Port:     <none>
    Command:
      /work-dir/peer-finder
    Args:
      -on-start=/init/on-start.sh
      -service=test-mongodb-replicaset
    State:          Running
      Started:      Thu, 21 Feb 2019 11:05:13 +0000
    Ready:          False
    Restart Count:  0
    Environment:
      POD_NAMESPACE:  test (v1:metadata.namespace)
      REPLICA_SET:    rs0
    Mounts:
      /data/configdb from configdir (rw)
      /data/db from datadir (rw)
      /init from init (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-f6p6g (ro)
      /work-dir from workdir (rw)
Containers:
  mongodb-replicaset:
    Container ID:  
    Image:         mongo:3.6
    Image ID:      
...
Conditions:
  Type              Status
  Initialized       False 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
...
Events:          <none>
```

**Comment by aylei** [comment]
Thanks! I will take a look this weekend.

**Comment by aylei** [comment]
Addressed in [0.0.2 release](https://github.com/aylei/kubectl-debug/releases/tag/0.0.2)
